components/admin_console/audits/audits.tsx

```diff
@@ -11,52 +10,47 @@ import LoadingScreen from 'components/loading_screen';
 
 import ReloadIcon from 'components/widgets/icons/fa_reload_icon';
 
-export default class Audits extends React.PureComponent {
-    static propTypes = {
-        isLicensed: PropTypes.bool.isRequired,
+type Props = {
+    isLicensed: boolean;
+    audits: Array<any>;
+    actions: {
+        getAudits: () => any;
```

Comment:
shouldn't this be ActionFunc instead of any?

Reply:
ActionFunc throws an error "Property 'then' does not exist on type 'ActionFunc'"

Reply:
I've asked a fellow team member about this, will report back once I have his feedback, sorry for the delay